UnityProject/Assets/Scripts/NPC/AI/MobExplore.cs

```diff
@@ -28,6 +29,9 @@ public enum Target
 			players
 		}
 
+		[Tooltip("The reagent used by emagged cleanbots")]
+		[SerializeField] private Reagent CB_REAGENT;
```

Comment:
```suggestion
		[SerializeField] private Reagent cbReagent;
```
Style for privates is camelCase

Reply:
I thought constant-like variables in development guidelines were asked to be in all caps like normal constants.

Reply:
uhmmm, the rest of the code handles serialized fields just like any other private, even when they are cashing a reference.

Reply:
Development standards says if a field is assignable in the editor just make it private and add [SerializeField]. Im just tryna stay as close to the document says to do stuff.

Reply:
The thing I'm refering to is the comment on No.6 in development standards. "this is not an actual constant, but is intialized in Start() and never modified afterwards so it should still be named using this convention"

Reply:
it is not a serializable field tho. It is fine, leave it as is. We should eventually move it somewhere else anyways because that reference is only relevant for the cleanbot from all the mobs that use MobExplore.
I really want to rework the way we do AI.

Reply:
Yeah, working with AI is a bit annoying as all bots and some mobs AI all come from the same script with the only distinguishing them being a couple switches and/or ifs